src/config/index.js

```diff
@@ -1,2 +1,5 @@
 // eslint-disable-next-line import/prefer-default-export
```

Comment:
Can you revert the changes to this file? The API_HOST(base_url) should be an environment variable.

Reply:
Done!